spring-cloud-azure-samples/pom.xml

```diff
@@ -29,6 +29,7 @@
         <module>spring-cloud-azure-servicebus-topic-binder-sample</module>
         <module>spring-cloud-azure-storage-queue-operation-sample</module>
         <module>spring-cloud-azure-storage-queue-integration-sample</module>
+        <module>azure-messaging-sample</module>
```

Comment:
nit: The package is already named as `spring-cloud-azure-messaging`. Maybe we can still use `spring-cloud-azure-messaging-sample` to keep consistent with other samples.

Reply:
I have plan to remove the long redundant spring-cloud prefix for all submodules, please refer #308